src/layout/nav/NavLink.tsx

```diff
@@ -15,19 +15,35 @@ import {classNames} from "../../utils";
 export type NavLinkProps = {
     iconShape?: string;
     className?: string;
+    type?: NavLinkType;
+    onClick?: (evt: React.MouseEvent<HTMLElement>) => void;
 };
 
+export enum NavLinkType {
+    stepNavLink = "btn btn-link clr-wizard-stepnav-link",
```

Comment:
Do we need to pass this value in from parent component?  Would it be better to just have a property in NavLinkProps?  For example, we could do something like `isWizardStep: boolean`?  Not a blocking comment -just a suggestion.

Reply:
@ben-schumacher Nice suggestion. I have change the code accordingly.